.meta/sinks/socket.toml

```diff
@@ -25,4 +25,4 @@ type = "string"
 common = true
 examples = ["92.12.333.224:5000"]
 required = true
-description = "The address to bind to."
+description = "The address to bind to, this must include some <host>:<port> pair."
```

Comment:
Can we not enforce this format during validation? This doesn't need to block the PR, but we should open an issue to do so if that is the correct way to handle this.

Reply:
We do this already by providing validation errors at _build_ time.

Reply:
The existing part of this is actually incorrect. We don't bind to this address, we connect to it.

Reply:
Ah yeah, we should update that.

Reply:
done in #1585